apps/api/src/document/document.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Controller, Get, Post, Body, Patch, Param, Delete, Put } from '@nestjs/common';
+import { Public } from 'nest-keycloak-connect';
+import { DocumentService } from './document.service';
+import { CreateDocumentDto } from './dto/create-document.dto';
+import { UpdateDocumentDto } from './dto/update-document.dto';
+
+@Controller('document')
+export class DocumentController {
+  constructor(private readonly documentService: DocumentService) { }
+
+  @Public()
```

Comment:
Please make sure that the controller is secured and only authenticated people are able to use the document controller.

Please remove the `@Public()` decorators of all actions